### ISSUE TITLE
Most of the timeline is missing

### ISSUE THREAD

**Issue by andybalaam** [vector-im/element-web#21432]
### Steps to reproduce

1. Enter a room
2. Notice there is an upward arrow to scroll up to unread messages
3. Scroll up, but there is nothing to see

#### What did you expect?
I know there are more messages in this room, and I expected to see them.

#### What happened instead?
I see only a couple of messages, and empty space above them.

![image](https://user-images.githubusercontent.com/76812/158571147-18c5d917-edb0-4815-851b-123e16fd296b.png)

When I press PageUp, the screen goes completely blank.

### Operating system

Ubuntu 21.10

### Browser information

Firefox 98.0

### URL for webapp

https://develop.element.io

### Application version

Element version: 1f97bc23256a-react-26e6f8deca1d-js-18943d6519e9 Olm version: 3.2.8

### Homeserver

matrix.org

### Will you send logs?

Yes

**Comment by dbkr** [comment]
I think I may have just had this too - I couldn't scroll up past a certain point in a room, with no indication that it was trying to paginate backwards for more messages. Eventually it loaded more scrollback, not sure what caused it to do so.

**Comment by jryans** [comment]
I wonder if perhaps we should enable `ScrollPanel`'s debug logging for develop temporarily, so that when people do hit these cases, we'll have some (hopefully relevant) logs...?

**Comment by novocaine** [comment]
I also got this same this morning

**Comment by novocaine** [comment]
possibly duplicate of https://github.com/vector-im/element-web/issues/16983

**Comment by turt2live** [comment]
if the history comes back in a bit, it might just be the decryption loop being slow. We decrypt enough messages to get the room list preview working, but lazily load the rest of the room over time.

Larger accounts see this behaviour more prominently.

**Comment by andybalaam** [comment]
The history did come back in a bit. The behaviour was confusing and scary if I cared about that history, so I think if it needs to be slow we should have a spinner or equivalent.

**Comment by turt2live** [comment]
I don't think it *needs* to be slow, but we should indeed have a loading spinner. Closing in favour of https://github.com/vector-im/element-web/issues/16983 though, which has a bit more context.

**Comment by turt2live** [comment]
oh, wrong issue, nevermind.

**Comment by MadLittleMods** [comment]
Reproduced this a couple more times but in my case the whole timeline went blank as I was scrolling down to catch up on messages and didn't recover.

![](https://user-images.githubusercontent.com/558581/159696723-f468c8b2-73f2-4b87-bdc9-f63bb5e07f7d.png)

Rageshake logs:

 - https://github.com/matrix-org/element-web-rageshakes/issues/11610
 - https://github.com/matrix-org/element-web-rageshakes/issues/11627

 Relevant logs for this kind of bug:

 - `Discarding an event, we don't have a call/peerConn m.room.message`
 - `Discarding an event, we don't have a call/peerConn m.room.encrypted`
 - `Discarding an event, we don't have a call/peerConn org.matrix.msc3381.poll.response`
 

> I wonder if perhaps we should enable `ScrollPanel`'s debug logging for develop temporarily, so that when people do hit these cases, we'll have some (hopefully relevant) logs...?

Created https://github.com/vector-im/element-web/issues/21532 to track adding a way to turn on this conditional debug logging while using the app instead of hard-coded into the build. And also useful for cases where we need more info from people and can just prompt to add more logging for those relevant components.

**Comment by JokerGermany** [comment]
I don't know it its related @t3chguy send me here:
I have the opposite problem:
enter the room and you see nothing, Notice there is an upward arrow to scroll up to unread messages, klick on it and than see posts.
After scrolling up, i can't scroll down, but there is a button to jump to the newest message and then i get the black screen back
https://youtu.be/5baXib_cBDQ

**Comment by JokerGermany** [comment]
In my case there is nothing missing, just additional black "spacer".
I asked others.
https://youtu.be/Fybb6YNs8g0

**Comment by JokerGermany** [comment]
oh and now i can not scroll up anymore, okay, now i have an missing timeline XD

**Comment by JokerGermany** [comment]
At least one person in #datenschutzraum:matrix.org has the same problem

**Comment by JokerGermany** [comment]
There is a fluffychat user who thinks that he has the same Problem.

**Comment by t3chguy** [comment]
This is causing e2e test flakes right now

![Uploading image.png…]()
